backup.go

```diff
@@ -154,17 +164,40 @@ func backup(ID string) error {
 	}
 	fmt.Printf("Creating backup of %s (%s, %s)\n", conf.Name[1:], conf.Config.Image, conf.ID[:12])
 
-	paths = []string{}
+	var mounts []types.MountPoint
+	var backupFiles []types.MountPoint
+
+	for _, mount := range conf.Mounts {
+		if mount.Type == "bind" {
+			skip := false
+			for _, exclude := range optExclude {
+				if strings.HasPrefix(mount.Source, exclude) {
```

Comment:
From a user's perspective, I wonder if we shouldn't support globbing here.

Reply:
I did wonder about that, either glob or regex. I'm not sure how common that need would be though, to exclude a more complex pattern of files in a bind (rather than entire folders/trees). The question then is, do you match glob/regex against the path to the bind, or individually against all the files searched in the bind?